packages/smooth_app/lib/pages/preferences/user_preferences_account.dart

```diff
@@ -377,16 +422,32 @@ class _UserPreferencesPageState extends State<UserPreferencesSection> {
           context: context,
         ),
         iconData,
+        userId: userId,
+        type: type,
+        language: language,
       );
 
   Widget _getListTile(
     final String title,
     final VoidCallback onTap,
-    final IconData leading,
-  ) =>
+    final IconData leading, {
+    final String? userId,
+    final UserProductSearchType? type,
+    final OpenFoodFactsLanguage? language,
+  }) =>
       UserPreferencesListTile(
         title: Text(title),
         onTap: onTap,
         leading: UserPreferencesListTile.getTintedIcon(leading, context),
+        trailing: (userId != null && type != null && language != null)
+            ? FutureBuilder<int?>(
+                future: _getMyCount(userId, type, language),
+                builder: (BuildContext context, AsyncSnapshot<int?> snapshot) {
+                  return Text(
```

Comment:
Could make sense to display a `CircularProgressIndicator` until the fetch is completed.

Reply:
I guess what @monsieurtanuki  meant was this,  @prathamsoni11 Just before line  427 add this
```
                   if (snapshot.connectionState != ConnectionState.done) {
                    return const SizedBox(
                        height: LARGE_SPACE,
                        width: LARGE_SPACE,
                        child: CircularProgressIndicator.adaptive());
                  }
```

Reply:
ok I will add this